intr.S

```diff
@@ -37,10 +37,11 @@ general_exception:
         beq     $k1, $k0, 1f
         nop
 
-        sll     $k0, $k0, 1            /* Multiply cause by two */
-        la      $k1, ge_table          /* Compute address of proper handler in ge table */
-        add     $k1, $k1, $k0
-        jr      $k1                    /* Jump to handler */ 
+        la      $k1, general_exception_table
+        sll     $k0, $k0, 2                           /* Multiply exc_code by 4 */  
+        add     $k1, $k1, $k0                         /* Add exc_code */  
+        lw      $k1, 0($k1)                           /* Compute address of proper handler in ge table */
```

Comment:
If an entry is `NULL` choose `kernel_oops` to avoid filling in `general_exception_table` in entries that are not handled.